lib/grppc.gi

```diff
@@ -1852,6 +1852,87 @@ InstallGlobalFunction( GapInputPcGroup, function(U,name)
 
 end );
 
+#############################################################################
+##
+#F  PrintPcPresentation( <grp>, <commBool> )  . . . . . . .  gap input string
+##
+##  Display the pc relations of a pc group.
+##  If commBool the commutator presentation is printed,
+##  otherwise the power presentation.
+##  Trivial commutators / powers are not printed.
+##  The generators  will  be  named "g<i>".
+##
+InstallGlobalFunction( PrintPcPresentation, function(G, commBool)
+    local pcgs, n, F, gens, i, pis, exp, t, h, rel, commPower, j;
+
+    pcgs:=Pcgs(G);
+    n:=Length(pcgs);
+    if n=0 then
+        Print("Trivial Group\n");
+        return;
+    fi;
+    F    := FreeGroup( n, "g" );
+    gens := GeneratorsOfGroup( F );
+    pis  := RelativeOrders( pcgs );
+
+    # the power
+    for i in [1..n] do
+        exp := ExponentsOfRelativePower( pcgs, i ){[i+1..n]};
+        t   := One( F );
+        for h in [i+1..n] do
+            t := t * gens[h]^exp[h-i];
+        od;
+        if t = One( F ) then
+            Print(gens[i], "^", pis[i], " = id\n");
+        else
+            Print(gens[i], "^", pis[i], " = ", t, "\n");
+        fi;
+    od;
+
+    # the commutators / powers
+    for i in [1..n] do
+        for j in [i+1..n] do
+            if commBool then
+                commPower := Comm( pcgs[j], pcgs[i] );
+            else
+                commPower := pcgs[j]^pcgs[i];
+            fi;
+            exp := ExponentsOfPcElement( pcgs, commPower ){[i+1..n]};
+            t   := One( F );
+            for h in [i+1..n] do
+                t := t * gens[h]^exp[h-i];
+            od;
+            if commBool then
+                if t = One( F ) then
+                    continue;
+                fi;
+                Print("[ ", gens[j], ", ", gens[i] , " ] = ", t, "\n");
+            else
+                if t = gens[j] then
+                    continue;
+                fi;
+                Print(gens[j], " ^ ", gens[i] , " = ", t, "\n");
+            fi;
+        od;
+    od;
+end );
+
+#############################################################################
+##
+#M  Display( <grp> )
+##
+InstallMethod( Display,
+    "for a pc group",
+    [ IsPcGroup ],
+    function( G )
+        if IsTrivial(G) then
+            Print("Trivial Group\n");
+            return;
+        fi;
+        Print("pc-group with relations:\n");
+        PrintPcPresentation( G, false );
```

Comment:
Some code in GAP uses spaces after opening parens, some not; I don't care much either way, but I'd recommend to at least stay consistent within the code of a single functions. So I'd suggest to adapt this call to the other code in this function:
```suggestion
        PrintPcPresentation(G, false);
```